packages/devtools_app/lib/src/memory/memory_heap_tree_view.dart

```diff
@@ -369,7 +366,7 @@ class HeapTreeViewState extends State<HeapTree> with AutoDisposeMixin {
       mainAxisAlignment: MainAxisAlignment.end,
       children: [
         Expanded(child: snapshotDisplay),
-        if (hasDetails) const SizedBox(width: defaultSpacing),
+        hasDetails ? const SizedBox(width: defaultSpacing) : const SizedBox(),
```

Comment:
why is this change necessary?

Reply:
I think initially it was because the line didn't fit in one line but it should fit in one line. Reverted.